libvast_test/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+## Configure data.hpp
+
+# Bro logs.
+set(test_data "#ifndef TEST_DATA\n#define TEST_DATA")
+set(test_data "${test_data}\n\nnamespace bro {\n")
+file(GLOB logs RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" artefacts/logs/bro/*)
+foreach (log ${logs})
```

Comment:
Everywhere else we don't add spaces after CMake function names.

Reply:
But `foreach` is a keyword. We are a bit inconsistent with spaces after keywords, and the code here is just moved from `libvast/CMakeLists.txt`. Lets make it more consistent in a different PR.